mix.exs

```diff
@@ -57,7 +57,9 @@ defmodule Surgex.Mixfile do
       {:ex_doc, "~> 0.23", only: :dev, runtime: false},
       {:ex_machina, "~> 2.3", only: :test},
       {:inch_ex, "~> 2.0", only: [:dev, :test]},
+      {:jabbax, "~> 1.0"},
```

Comment:
Surgex is a mixed bag of helper modules - some aren't related to web APIs at all. A common pattern with optional dependencies is to only define the modules that require them if they're available - using [Code.ensure_loaded?/1](https://hexdocs.pm/elixir/1.15.5/Code.html#ensure_loaded?/1). I think that would be a better approach as it allows for using `surgex` without requiring and compiling `jabbax` if it's not necessary.

Reply:
We actually use that approach in `Surgex.DateTime` and `Surgex.Changeset`.

Reply:
Okay, so I'm now using `Code.ensure_loaded?` as well. I've also bumped the version and updated the CHANGELOG.